library/src/main/java/com/getstream/sdk/chat/model/Channel.java

```diff
@@ -369,6 +369,9 @@ public Channel copy() {
         if (channelState != null) {
             clone.channelState = channelState.copy();
         }
+        if (!extraData.isEmpty()) {
+            clone.extraData = extraData;
```

Comment:
I think you need to do at least a shallow-copy here or otherwise the diff will always be empty